packages/core/src/checkout/checkout-action-creator.ts

```diff
@@ -29,11 +29,14 @@ export default class CheckoutActionCreator {
                 of(createAction(CheckoutActionType.LoadCheckoutRequested)),
                 merge(
                     this._configActionCreator.loadConfig({
-                        ...options,
```

Comment:
We should be explicit about what parameters to pass into `ConfigActionCreator` and `FormFieldActionCreator` otherwise we could be passing more than needed. For instance, `params: { include: ['cart.lineItems.physicalItems.categoryNames', 'cart.lineItems.digitalItems.categoryNames'] }` is only intended for the checkout endpoint, not the settings or form fields endpoint.